app/src/views/TermsAndConditions/index.tsx

```diff
@@ -0,0 +1,416 @@
+                                                        },
+                                                    )}
+                                                </p>
+                                            </li>
+                                        </ul>
+                                    </li>
+                                    <li>{strings.userRegistrationDescriptionList3}</li>
+                                    <li>{strings.userRegistrationDescriptionList4}</li>
+                                </ul>
+                                <div>{strings.userRegistrationDescription5}</div>
+                            </>
+                        )}
+                    />
+                    <Container
+                        heading={strings.unAuthorisedUsesHeading}
+                        withHeaderBorder
+                        withInternalPadding
+                        containerRef={unauthorisedUsersRef}
+                        footerContentClassName={styles.headerDescription}
+                        footerContent={(
+                            <>
+                                <p>{strings.unAuthorisedUsesDescription}</p>
+                                <ul>
+                                    <li>{strings.unAuthorisedUsesDescriptionList1}</li>
+                                    <li>{strings.unAuthorisedUsesDescriptionList2}</li>
+                                    <li>{strings.unAuthorisedUsesDescriptionList3}</li>
+                                    <li>{strings.unAuthorisedUsesDescriptionList4}</li>
+                                    <li>{strings.unAuthorisedUsesDescriptionList5}</li>
+                                    <li>{strings.unAuthorisedUsesDescriptionList6}</li>
+                                    <li>{strings.unAuthorisedUsesDescriptionList7}</li>
+                                    <li>{strings.unAuthorisedUsesDescriptionList8}</li>
+                                    <li>{strings.unAuthorisedUsesDescriptionList9}</li>
+                                    <li>{strings.unAuthorisedUsesDescriptionList10}</li>
+                                </ul>
+                                <p>{strings.unAuthorisedUsesDescription2}</p>
+                                <p>{strings.unAuthorisedUsesDescription3}</p>
+                            </>
+                        )}
+                    />
+                    <Container
+                        heading={strings.complianceDateOwnerHeading}
+                        withHeaderBorder
+                        withInternalPadding
+                        containerRef={complianceOfUserOwnerRef}
+                        footerContentClassName={styles.headerDescription}
+                        footerContent={(
+                            <>
+                                <p>{strings.complianceDateOwnerDescription1}</p>
+                                <p>
+                                    {!accepted && resolveToComponent(
```

Comment:
We should move the placement of the conditional checks. We most probably don't want to render empty <p> tags